docs/pact-functions.md

```diff
@@ -352,7 +352,7 @@ Special form binds to a yielded object value from the prior step execution in a
 
 ### reverse {#reverse}
 
-*l*&nbsp;`[<a>]` *&rarr;*&nbsp;`[<a>]`
+*list*&nbsp;`[<a>]` *&rarr;*&nbsp;`[<a>]`
```

Comment:
great catch!

Reply:
I changed it in the `.hs` file itself, therefore this is just the Docgen'd rendering.